### PR TITLE
figures 2 and 3 referenced to cartoons

### PR DESCRIPTION
**Pull request by Farnazmdi**

### CODE REVIEW

**Comment by Farnazmdi** [comment]
This is what I meant.

**Comment by jclagarde** [comment]
> This is what I meant @jclagarde.

@Farnazmdi Ohh okay! Yeah, I believe my branch wasn't updated either. Thank you!

File: manuscript/03.results.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ In this section, the interface of the model and possible input/outputs are expla
...
   
 
-![**The interface of tHMM.** (a) The input data gets converted to the lineage objects with cells instantiated with properties from the data, such as cells' observations, paret-daughter link, generation number, etc. (b) The fitting process includes expectation and maximization steps. (c) Outputs of the model after fitting including the tree of states, the tree of observations, the transition probabilities, and the most likely distribution parameters.](./output/figure2.svg){#fig:2}
+![**The interface of tHMM.** (a) The input data gets converted to the lineage objects with cells instantiated with properties from the data, such as cells' observations, paret-daughter link, generation number, etc. (b) The fitting process includes expectation and maximization steps. (c) Outputs of the model after fitting including the tree of states, the tree of observations, the transition probabilities, and the most likely distribution parameters.](./figures/cartoons/figure2.svg){#fig:2}
```
Comment (aarmey) [inline manuscript/03.results.md:22]: Check that this shows up in the manuscript. If not, ping me and I'll try and fix this.

Comment (Farnazmdi) [reply manuscript/03.results.md:22]: Thank you, will do.

**Comment by Farnazmdi** [comment]
@aarmey sorry, may I cancel the builds of the other project that is running on the Jenkins (resistance ms?) except for their last commit? Its build takes more than an hour I think...

**Comment by aarmey** [comment]
Check with Marc.

File: manuscript/03.results.md (hunk 1 above)
Comment (Farnazmdi) [reply manuscript/03.results.md:22]: They didn't show up @aarmey

Comment (aarmey) [reply manuscript/03.results.md:22]: Ok, I just added a command to copy them to the output folder, so revert this to the original path within the makefile and then they should work.

Comment (Farnazmdi) [reply manuscript/03.results.md:22]: Will do, thank you.

**Pull request merged**